soh/src/code/z_parameter.c

```diff
@@ -3388,6 +3388,9 @@ void Interface_UpdateMagicBar(PlayState* play) {
 
         default:
             gSaveContext.magicState = MAGIC_STATE_IDLE;
+            if (CVarGetInteger(CVAR_COSMETIC("Consumable.MagicBorder.Changed"), 0)) {
+                sMagicBorder = CVarGetColor24(CVAR_COSMETIC("Consumable.MagicBorder.Value"), sMagicBorder_ori);
+            }
```

Comment:
In the other spots we have an `else` to set back the original colors, we probably need that here to make sure the color can reset.

Since `_IDLE` comes the next frame after `_RESET`, I think we can restore `_RESET` back to its vanilla state and just have the color override handling here.